packages/builtin/array.pony

```diff
@@ -276,16 +276,24 @@ class Array[A] is Seq[A]
 
     this
 
-  fun find(value: A!, offset: USize = 0, nth: USize = 0): USize ? =>
+  fun find(value: A!, offset: USize = 0, nth: USize = 0,
+    predicate: {(box->A!,box->A!): Bool} val =
+      lambda (l:box->A!,r:box->A!):Bool => l is r end): USize ?
```

Comment:
Can you try to normalize the whitespace style to match the style found in the rest of the codebase?

For example, no space between `lambda` and the opening `(`, one space between a `,` and the next item in parameter lists, one space between the `:` and the following type, etc.